monai/bundle/properties.py

```diff
@@ -216,3 +221,42 @@
         BundlePropertyConfig.REF_ID: f"evaluator{ID_SEP_KEY}key_val_metric",
     },
 }
+
+MetaProterties = {
+    "version": {
+        BundleProperty.DESC: "bundle version",
+        BundleProperty.REQUIRED: False,
```

Comment:
I think verison should be required property, what do you think?

Thanks.

Reply:
Have updated it in https://github.com/Project-MONAI/MONAI/pull/6835/commits/63d3a845b92a643773e08a05482b939bbb88e20d.

Reply:
Hi @Nic-Ma, “meta_file” is not required, do we also have to make the bundle version required?
https://github.com/Project-MONAI/MONAI/blob/2daabf917a1e6afbc738d22db602f4859672a90d/monai/bundle/workflows.py#L222

It may affect `check_properties` since `MetaProterties` is automatically added to the property when initializing a bundleworflow.
https://github.com/Project-MONAI/MONAI/blob/6938718b757e0366b8fc6344a1e7dbafcf588bbd/monai/bundle/workflows.py#L161